cloudflare.php

```diff
@@ -468,6 +469,16 @@ function purgeCache()
     }
 }
 
+function sslRewrite() {
+    if ( isset( $_SERVER['HTTP_X_FORWARDED_PROTO'] ) && $_SERVER['HTTP_X_FORWARDED_PROTO'] == 'https' ) {
+        $_SERVER['HTTPS'] = 'on';
+
+        return true;
+    }
+
+    return false;
```

Comment:
Also if we were to test `sslRewrite()` it would need to return `true` or `false` so we could `assert` whether it exited correctly.